src/libcore/char.rs

```diff
@@ -119,16 +119,16 @@ pub fn from_u32(i: u32) -> Option<char> {
 /// ```
 #[inline]
 #[unstable(feature = "core", reason = "pending integer conventions")]
-pub fn from_digit(num: uint, radix: uint) -> Option<char> {
+pub fn from_digit(num: u32, radix: u32) -> Option<char> {
```

Comment:
See the guideline here #22240

Actually `u32` here is not as arbitrary as it may seem, radixes, for example, are directly tied to the textual representation of numbers, and `u32` can be viewed as value of unicode code point. And the numeric values of unicode digits (e.g. Roman numerals) can be larger than  256 (not in this example, but in others). Regardless, I think `u8` would be as good as `u32` and the choice doesn't really matters in practice :)